lib/prepareStepOfflineInvoke.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const _ = require('lodash');
+const path = require('path');
+
+/**
+ * Special settings for use with serverless-offline.
```

Comment:
This should be `step-functions-offline` instead of `serverless-offline`.

Reply:
Fixed